Archiver duration timer improvements

ChrisKujawa opened the pull request:
## Description

This PR iterates over the archiving timers/duration metrics and their panels introduced with https://github.com/camunda/camunda/pull/30684

 

 * Combine timers under one metric and rename them
   * Make it more clear that we observing the duration and add
     tags for better separations
   * Add histogram configuration, so we have more details in the
     metric
* Add new metric to observe the complete archiving duration
  * This is combining the search, reindex and deletion, might
     be useful in future investigations
* Reiterate over timer panels
  * Make use of histograms instead of just showing avg's
  * Show complete archiving duration and separated ones,
   search, reindex and delete
  * Add descriptions to panels
  * Remove unused panels

New Panels:

![archiving-durations](https://github.com/user-attachments/assets/9ab2d901-25e7-4629-8994-0f5f86ba0871)

<!-- Describe the goal and purpose of this PR. -->

related #30684 
covering review hint https://github.com/camunda/camunda/pull/30684#discussion_r2033137038

entangled90 reviewed (commented):
Is the screenshot in the description up to the date with the grafana dashboard?
Because I see two charts in the bottom row have the same title, I assume one of them should be the one related to delete operation.

ChrisKujawa commented:
> Is the screenshot in the description up to the date with the grafana dashboard? Because I see two charts in the bottom row have the same title, I assume one of them should be the one related to delete operation.

@entangled90 good catch! I could swear I changed this yesterday 🤔 

Fixed now with https://github.com/camunda/camunda/pull/30830/commits/9fcd3359831192c751fc038fb8fcbb3ae99a5881!

entangled90 reviewed (approved):
:rocket:
